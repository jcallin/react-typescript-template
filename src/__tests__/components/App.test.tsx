import React from "react";
import { shallow } from "enzyme";
import App from "../../components/App";

describe("App", () => {
  it("renders without crashing", () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
  });
});
