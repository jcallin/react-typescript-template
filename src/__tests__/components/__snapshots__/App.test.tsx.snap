// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Edit 
      <code>
        src/components/App.tsx
      </code>
       and save to reload.
    </p>
  </header>
</div>
`;
